setup.py

```diff
@@ -1,6 +1,47 @@
-from distutils.core import setup
+import io
+import os
 
-setup(name='pysize',
-      version='1.0',
-      py_modules=['pysize'],
-      )
\ No newline at end of file
+from setuptools import find_packages, setup
+
+NAME = 'pysize'
+DESCRIPTION = "Use to quickly measure the size of your python objects."
+VERSION = None
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+try:
+    with io.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
+        long_description = '\n' + f.read()
+except FileNotFoundError:
+    long_description = DESCRIPTION
+
+about = {}
+if not VERSION:
+    with open(os.path.join(here, NAME, '__version__.py')) as f:
+        exec(f.read(), about)

```

Comment:
No, it's a horrible idea and not related to what I asked. https://github.com/pypa/setuptools_scm